Custom dialog using <dialog> element #9

Rahul-Palamarthi opened the pull request:
completed Dialog component

pooranjoyb reviewed (commented):
Great job, @Rahul-Palamarthi! Thanks for creating the pull request!<p>Soon the maintainers/owner will review it and provide you with feedback/suggestions.</p>

pooranjoyb reviewed (changes requested):
Hey @Rahul-Palamarthi 
Please make sure that you add the Dialog in the Components section, i.e in the `/component` route and not in the landing page.

Rahul-Palamarthi commented:
> Hey @Rahul-Palamarthi 
> Please make sure that you add the Dialog in the Components section, i.e in the `/component` route and not in the landing page.

Ok I will change it

Rahul-Palamarthi commented:
> Hey @Rahul-Palamarthi Please make sure that you add the Dialog in the Components section, i.e in the `/component` route and not in the landing page.

@pooranjoyb I have removed the Dialog from landing page and added in the components page 🚀
